src/domain/templates/components/TemplatesAdmin/TemplatesAdmin.tsx

```diff
@@ -122,18 +123,23 @@ const TemplatesAdmin = ({
 
   // Update Template
   const [editTemplateMode, setEditTemplateMode] = useState(alwaysEditTemplate);
-  const [updateTemplate] = useUpdateTemplateMutation({
-    refetchQueries: ['AllTemplatesInTemplatesSet', 'TemplateContent'],
-  });
-  const [updateCallout] = useUpdateCalloutTemplateMutation();
-  const [updateCommunityGuidelines] = useUpdateCommunityGuidelinesMutation();
+
+  const refetchQueries = ['AllTemplatesInTemplatesSet', 'TemplateContent'];
+  const [updateTemplate] = useUpdateTemplateMutation({ refetchQueries });
+  const [updateCallout] = useUpdateCalloutTemplateMutation({ refetchQueries });
+  const [updateCommunityGuidelines] = useUpdateCommunityGuidelinesMutation({ refetchQueries });
+  const [updateTemplateFromCollaboration] = useUpdateTemplateFromCollaborationMutation({ refetchQueries });
 
   const handleTemplateUpdate = async (values: AnyTemplateFormSubmittedValues) => {
     if (!selectedTemplate) {
       return;
     }
-    const { updateTemplateVariables, updateCalloutVariables, updateCommunityGuidelinesVariables } =
-      toUpdateTemplateMutationVariables(templateId!, selectedTemplate, values);
+    const {
+      updateTemplateVariables,
+      updateCalloutVariables,
+      updateCommunityGuidelinesVariables,
+      updateCollaborationTemplateVariables,
+    } = toUpdateTemplateMutationVariables(templateId!, selectedTemplate, values);
```

Comment:
Ahhh, I thought we're done with these try/catch suggestions. It was communicated with you many times and it's part of the instructions in the .coderabbit.yaml file. 
Please, reduce the noise around error handling as we have a global handler for all queries/mutations.

Reply:
@coderabbitai, is it possible that you are applying the learnings only to the specific file that we commented about? or you take into account the learnings to then apply them to all the PRs in all the files?

Reply:
@coderabbitai can you create a new PullRequest modifying our .coderabbit.yaml to avoid unnecessary alerts about missing try catches and error handling where the errors are handled by the global error handler?